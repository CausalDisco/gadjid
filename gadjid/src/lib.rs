// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
