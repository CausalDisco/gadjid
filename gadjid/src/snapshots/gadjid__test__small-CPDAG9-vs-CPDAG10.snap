---
source: src/lib.rs
expression: "test(\"20009.CPDAG-10\", \"20010.CPDAG-10\")"
---
g_true: 20009.CPDAG-10
g_guess: 20010.CPDAG-10
ts:
  - 3
  - 9
  - 2
  - 4
  - 3
ys:
  - 5
  - 6
  - 2
  - 3
  - 4
zs:
  - []
  - []
  - []
  - - 0
    - 5
  - []
nams:
  - - 0
    - 1
    - 2
    - 4
    - 5
    - 6
    - 7
    - 8
    - 9
  - - 0
    - 1
    - 2
    - 3
    - 4
    - 5
    - 6
    - 7
    - 8
  - - 0
    - 1
    - 3
    - 4
    - 5
    - 6
    - 7
    - 8
    - 9
  - - 0
    - 1
    - 2
    - 3
    - 5
    - 6
    - 7
    - 8
    - 9
  - - 0
    - 1
    - 2
    - 4
    - 5
    - 6
    - 7
    - 8
    - 9
nvas:
  - - 0
    - 1
    - 2
    - 4
    - 5
    - 6
    - 7
    - 8
    - 9
  - - 0
    - 1
    - 2
    - 3
    - 4
    - 5
    - 6
    - 7
    - 8
  - - 0
    - 1
    - 3
    - 4
    - 5
    - 6
    - 7
    - 8
    - 9
  - - 0
    - 1
    - 2
    - 3
    - 5
    - 6
    - 7
    - 8
    - 9
  - - 0
    - 1
    - 2
    - 4
    - 5
    - 6
    - 7
    - 8
    - 9
pa_true_1st_T: []
an_true_1st_T:
  - 3
ch_true_1st_T: []
de_true_1st_T:
  - 3
poss_de_true_1st_T:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
prop_an_true_1st_T_and_1st_Y:
  - 5
shd:
  - 0.3333333333333333
  - 15
oset_aid:
  - 0.5666666666666667
  - 51
parent_aid:
  - 0.5666666666666667
  - 51
ancestor_aid:
  - 0.5666666666666667
  - 51
